.github/workflows/spark_sql_test_ansi.yml

```diff
@@ -44,8 +42,8 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest]
-        java-version: [11]
-        spark-version: [{short: '3.4', full: '3.4.2'}]
+        java-version: [17]
+        spark-version: [{short: '4.0', full: '4.0.0-preview1'}]
```

Comment:
Is this particular for ansi test? Hmm, is there any difference between 3.4 and 4.0?

Reply:
This has been disabled. List of failing tests https://github.com/apache/datafusion-comet/issues/551

Reply:
Hmm, before we can enable it, can we still use 3.4 to run these tests?

Reply:
I have not tried running this with 3.4, but based on the comment, it likely fails.

Reply:
Ah, I forgot we don't run ansi test in CI.